portal-kernel/src/com/liferay/document/library/kernel/util/PDFProcessor.java

```diff
@@ -40,6 +40,8 @@ public void generateImages(
 			FileVersion sourceFileVersion, FileVersion destinationFileVersion)
 		throws Exception;
 
+	public void generatePDFPreviews(long userId);
```

Comment:
One minor (and optional) thing here. Why not name the method simply `generateImages` to keep symmetry with the other one? This does basically the same thing, but for all company documents.

Reply:
I didn't like use the name the same (to make it easier to know that this methods do different things).

I think that "generatePDFPreviews" is more accurate

To keep symetry,  instead of mame both "generateImages" i'd prefer rename it to  "generatePDFPreview" in singular